packages/@jsii/integ-test/utils/index.ts

```diff
@@ -1,84 +1,78 @@
+import { Readable } from 'stream';
+import { createWriteStream } from 'fs';
 import * as cp from 'child_process';
 import * as https from 'https';
-import * as path from 'path';
-import { PathLike, promises as fs } from 'fs';
-
-export const minutes = (num: number) => num * 1000 * 60
+import { IncomingMessage } from 'http';
 
 /**
- * rmdirRecursive
+ * @param num a quantity of minutes (could be fractional)
  *
- * recursive directory removal for cleanup after build test. Node10 fs module
- * doesn't support the `recursive` option
+ * @return equivalent number of milliseconds
  */
-export const rmdirRecursive = async (dir: PathLike) => {
-  const contents = await fs.readdir(dir);
-  await Promise.all(contents.map(async (file) => {
-    const currentPath = path.join(dir.toString(), file);
-    if ((await fs.lstat(currentPath)).isDirectory()) {
-      await rmdirRecursive(currentPath);
-    } else {
-      await fs.unlink(currentPath);
-    }
-  }));
-
-  await fs.rmdir(dir);
-};
+export function minutes(num: number): number {
+  return num * 1000 * 60;
```

Comment:
I still think we should skip the runtime multiplication that can be done upfront (`60_000`). But whatevs.

Reply:
my thinking here is if we continue to add long running integ-tests having this util may be handy?

Reply:
I mean, sure, but you're still doing the `1000 * 60` multiplication at run-time, when it's a fairly obvious value... to hard-code beforehand?